.github/workflows/release-create-release-candidate-branch.yml

```diff
@@ -23,7 +23,7 @@ jobs:
           git config --global user.email 'ott-release-script@jwplayer.com'
           git fetch origin ${{ github.ref_name }}
           git merge origin/${{ github.ref_name }}
-          yarn && yarn i18next
+          yarn && yarn web i18next
```

Comment:
`yarn web` is a shortcut for `yarn --cwd platforms/web`, which I believe is the same as running `yarn workspace @jwp/ott-web`.

The explicit `yarn workspace` command might be clearer for users familiar with yarn workspaces...

Reply:
Yes, that was my thinking. I got excited when I first read this, because I though yarn had built some sort of intelligent workspace picker, but then I dug down to find the `yarn web` implementation.

Reply:
See the discussion here: https://github.com/jwplayer/ott-web-app/pull/456#discussion_r1517766999

Do you suggest keeping it out of the documentation and only having it available for convenience?

Reply:
The point I had was about helping people interested in the web app to launch it. In this case you won't need to dig into docs file packages and platforms. Just use these commands and see the app. I don't mind it being `yarn workspace @jwp/ott-web` or `yarn start:web` in the docs if it could improve readability.